libs/wingc/src/type_check.rs

```diff
@@ -2483,59 +2495,77 @@ impl<'a> TypeChecker<'a> {
 		// the statement that initiated the bring, if any
 		stmt: Option<&Stmt>,
 	) {
-		let mut wingii_types = wingii::type_system::TypeSystem::new();
-
-		// Loading the SDK is handled different from loading any other jsii modules because with the SDK we provide an exact
-		// location to locate the SDK, whereas for the other modules we need to search for them from the source directory.
-		let assembly_name = if library_name == WINGSDK_ASSEMBLY_NAME {
-			// in runtime, if "WINGSDK_MANIFEST_ROOT" env var is set, read it. otherwise set to "../wingsdk" for dev
-			let manifest_root = std::env::var("WINGSDK_MANIFEST_ROOT").unwrap_or_else(|_| "../wingsdk".to_string());
-			let wingii_loader_options = wingii::type_system::AssemblyLoadOptions {
-				root: true,
-				deps: false,
-			};
-			let assembly_name = match wingii_types.load(manifest_root.as_str(), Some(wingii_loader_options)) {
-				Ok(name) => name,
-				Err(type_error) => {
-					self.type_error(TypeError {
-						message: format!("Cannot locate Wing standard library (checking \"{}\"", manifest_root),
-						span: stmt.map(|s| s.span.clone()).unwrap_or_default(),
-					});
-					debug!("{:?}", type_error);
-					return;
-				}
-			};
-
-			assembly_name
+		let jsii = if let Some(jsii) = self
+			.jsii_imports
+			.iter()
+			.find(|j| j.assembly_name == library_name && j.alias.name == alias.name)
+		{
+			jsii
```

Comment:
Checking my understanding - so in this first branch, we've found a module from `self.jsii_imports` that we've already imported it before, but because it might have only been a partial import, we still need to call `JsiiImporter` and try importing it again, yeah?

Reply:
Wait, I remember why this is here now: For the LSP case where we carry forward the vector of import data. In that case, we need to update the current symbol environment with the same info again.

The double-import thing is still an issue though